gnd/src/main/java/com/google/android/gnd/ui/map/gms/GoogleMapsMapAdapter.java

```diff
@@ -321,13 +320,17 @@ private void addMapPolyline(MapPolygon mapPolygon) {
   }
 
   private boolean isPolygonCompleted(List<Point> vertices) {
-    return vertices.size() != stream(vertices).distinct().count();
+    return vertices.get(vertices.size() - 1) == vertices.get(0);
   }
 
   private BitmapDescriptor bitmapDescriptorFromVector() {
     Drawable vectorDrawable = ContextCompat.getDrawable(context, R.drawable.ic_endpoint);
+    // Specify a bounding rectangle for the Drawable.
     vectorDrawable
-        .setBounds(4, 4, vectorDrawable.getIntrinsicWidth(), vectorDrawable.getIntrinsicHeight());
+        .setBounds((int) context.getResources().getDimension(R.dimen.polyline_endpoint_bound),
```

Comment:
Can we just trim these pixels off the vector instead of clipping them here?

Reply:
reduced the size of svg to 16px so now no clipping required.